tox.ini

```diff
@@ -1,19 +1,30 @@
 [tox]
 minversion=2.3.1
 envlist =
-    py{27,35}-ansible22-{pylint,unit,flake8,yamllint,generate_validation}
+    py{27,35}-ansible22-lint
+    py{27,35}-ansible22-unit
 skipsdist=True
 skip_missing_interpreters=True
 
 [testenv]
 deps =
+    wheel
```

Comment:
Maybe this could also be in `requirements.txt`, so that it broadens the audience benefited by having packages installed via wheel?